recipes/ipydatetime/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ipydatetime" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b083e08efef140c29091fe114ea2921f88989e56f0b5eecdde9ac0157bb0b10
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - jupyter-packaging
+    - nodejs
+    - python
+    - pip
+  run:
+    - ipywidgets >=7.0.0
+    - jupyter-packaging
+    - jupyterlab >=2
+    - nodejs
+    - python
+    - tzlocal
+
+test:
+  commands:
+    - pip check
+  requires:
+    - pip
```

Comment:
it's worth adding a check for whether the labextension is detected, see https://conda-forge.org/docs/maintainer/knowledge_base.html#jupyterlab-extensions